src/lammpsio/data.py

```diff
@@ -267,6 +275,8 @@ def read(self):
                     box_bounds[2], box_bounds[5] = [float(x) for x in line.split()[:2]]
                 elif "xy xz yz" in line:
                     box_tilt = [float(x) for x in line.split()[:3]]
+                elif "bonds" in line:
+                    bonds = Bonds(N=line.split()[0])
```

Comment:
Do you need to cast this to an int?
```suggestion
                    N_bonds = int(line.split()[0])
                    bonds = Bonds(N=N_bonds)
```

Reply:
Currently, `bonds` does not get saved into the `Snapshot`. We need to add a `Snapshot.bonds` property.

Then, either we can also add a `num_bonds` to the `Snapshot` constructor that makes this `Bonds` group in the first place (read only), or we give it a setter so we can assign a `Bonds` group. I think either could make sense, but the second one keeps the Snapshot constructor cleaner.